src/Query.sql

```diff
@@ -75,38 +71,70 @@ Query- select SNAME from Salespeople where SNUM IN (select SNUM from Orders);
        | Riffikn |
        +---------+
 
-
 6) List names of all customers matched with the salespeople serving them.
 --------------------------------------------------------------------------
-Query- select CNAME from Customers where SNUM IN (select SNUM from Salespeople);
-       Result Empty.
-
+Query- SELECT CNAME FROM customers WHERE SNUM IN (SELECT SNUM FROM salespeople);
```

Comment:
Display the name of salespeople as well.

Reply:
Done